docker/nginx/docker-entrypoint.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -eo pipefail
+
+if [ "$1" = 'nginx' ]; then
```

Comment:
Yup, this one is checking to see if the app is being invoked as `root` user.  With NGinx, the master thread by design gets started with `root`, and then spawns subprocesses as `www-data`